.travis.yml

```diff
@@ -13,14 +13,26 @@ matrix:
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION scalafmtCheck scalafmtSbtCheck test:compile test:doc
         - travis_wait 30 sleep 1800 &
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION test
-    - scala: 2.12.10
+    - scala: 2.12.11
       jdk: openjdk8
       script:
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION test:compile test:doc
         - travis_wait 30 sleep 1800 &
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION coverage test coverageReport
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION coverageAggregate
       after_success: sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION coveralls
+    - scala: 2.12.10
+      jdk: openjdk8
+      script:
+        - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION test:compile test:doc
+        - travis_wait 30 sleep 1800 &
+        - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION test
+    - scala: 2.13.2
```

Comment:
It looks like this particular one is (sometimes?) causing errors... kicking the build again...

<img width="1309" alt="Screenshot 2020-05-14 at 16 04 08" src="https://user-images.githubusercontent.com/914805/81903727-23510c00-95fd-11ea-9b5c-be2790b559df.png">